### PR TITLE
cleanup(pipeline) remove deprecated code for prefix and postfix

### PR DESCRIPTION
**Pull request by dduportal**
This PR removes the deprecated code for "prefix" and "postfix" (which have been replaced by transformers more than 1 year ago).

Depends on #590 (as there is a common change on the signature of a call to `ScmHandler.Init()`)

Associated documentation PR: https://github.com/updatecli/website/pull/264

## Test

To test this pull request, you can run the following commands:

```shell
make test
```

### CODE REVIEW

**Review by lemeurherve — APPROVED** [review]
🧹👏

**Comment by olblak** [comment]
I need to double check but I think prefix and postfix are still "used" in manifest example and documentation

**Comment by olblak** [comment]
I mean we should be sure to remove anu occurrence of this deprecated syntax

**Comment by dduportal** [comment]
> I mean we should be sure to remove anu occurrence of this deprecated syntax

Already done in https://github.com/updatecli/website/pull/264 :)

**Comment by dduportal** [comment]
- I'm waiting for review from 2 maintainers on this one (to be really sure that I did not do something wrong).
- Proposed milestone: 0.22.3

**Review by lemeurherve — DISMISSED** [review]

File: pkg/plugins/scms/git/scm.go
Comment (dduportal) [reply pkg/plugins/scms/git/scm.go:99]: @olblak this change is because this PR depends on https://github.com/updatecli/updatecli/pull/590. Do you mind switching the review to #590 first?

Comment (olblak) [reply pkg/plugins/scms/git/scm.go:99]: @dduportal Yup yup, I noticed that after my comment :p

**Comment by dduportal** [comment]
Last one \o/

**Review by lemeurherve — APPROVED** [review]

**Review by olblak — APPROVED** [review]
Glad to see old code going away which makes me realize how much updatecli grew since the beginning of the project

**Pull request merged**
